phoenix-pherf/pom.xml

```diff
@@ -133,10 +133,6 @@
       <groupId>org.apache.phoenix.thirdparty</groupId>
       <artifactId>phoenix-shaded-commons-cli</artifactId>
     </dependency>
-    <dependency>
-      <groupId>commons-io</groupId>
-      <artifactId>commons-io</artifactId>
-    </dependency>
```

Comment:
This dependency is removed because it is a duplicate of another one at line 125.
Let me know if it should be in a separate PR!

Reply:
Yes, please open a separate JIRA for this.